doc/generic/pgf/pgfmanual-en-pgfcalendar.tex

```diff
@@ -365,6 +392,8 @@ \subsection{Typesetting Calendars}
             two digits with a leading zero, if necessary).
         \item |\pgfcalendarcurrentday| The day of month of the current date
             (always two digits).
+        \item |\pgfcalendarcurrentweek| The week of the current date (always
+            two digits).
```

Comment:
```suggestion
            two digits, i.e.\ $5 \to 05$).
```

Reply:
Why only for `\pgfcalendarcurrentweek` and not for `…day` and `…month`, too?